cmd/template/cluster/provider/capo.go

```diff
@@ -81,7 +81,7 @@ func WriteOpenStackTemplateAppCR(ctx context.Context, config ClusterCRsConfig) e
 
 	clusterAppConfig := templateapp.Config{
 		AppName:   config.Name,
-		Catalog:   "control-plane-catalog",
+		Catalog:   config.ClusterAppCatalog,
```

Comment:
Do we really need to make it all configurable? I mean it should be pretty simple to change the manifest after this command is run. I would personally just set the catalog here to `giantswarm` and keep the number of flags limited from a UX pov

Reply:
I agree about not needing other catalogs, but we do need the ability to specify `giantswarm-test` to test branches of `cluster-openstack` and `default-apps-openstack` before they are released.

Reply:
On the UX side, we are hiding all OpenStack flags for now. Even when we un-hide them, we can leave these hidden as they are only really needed for GS internal use.